resources/views/layouts/create.blade.php

```diff
@@ -1,25 +1,29 @@
-@extends('layouts.app')
+@extends('layouts.admin')
 
-@section('header')
+@section('article_list')
     <h1>新規投稿</h1>
 @endsection
 
-@section('content')
-    <form action="{{route('article.store')}}" method="post">
-        @csrf
-        <ul>
-            <li>
-                <input type="text" name="title" id="title" placeholder="記事タイトル" required autofocus>
-            </li>
-            <li>
-                <textarea name="content" placeholder="ここに内容を入力してください。" rows="5" required></textarea>
-            </li>
-            <li>
-                <div>
-                    <button type="submit">投稿</button><br>
-                    <button type="button" onclick="location.href='{{route('article.create')}}'">戻る</button>
-                </div>
-            </li>
-        </ul>
-    </form>
+@section('article_post')
```

Comment:
dot notation
```suggestion
@section('article.post')
```

postはHTTP動詞のpostだと思うけど、POSTに相当する操作が複数ありえるから、HTTP動詞ではなくリソースの操作を表した方がいい（例：show, destroy, create, updateなど）

ので、新規作成の場合はこっちの方がいいと思う

```suggestion
@section('article.create')
```

Reply:
修正済。

Reply:
@DJMasaru Pushできてないと思う